src/dune_pkg/opam_repo.mli

```diff
@@ -22,6 +22,7 @@ val of_opam_repo_dir_path
   -> t
 
 val repo_id : t -> Repository_id.t option
+val source : t -> string option
```

Comment:
We don't have a type for sources?

Reply:
We do not, this is probably why this was hidden to begin with.